Cargo.toml

```diff
@@ -121,30 +121,30 @@ uuid = { version = "1.11", features = ["v4"] }
 parking_lot = "0.12"
 
 #alloy
-alloy-json-rpc = { version = "0.7", default-features = false }
-alloy-network = { version = "0.7", default-features = false }
-alloy-node-bindings = { version = "0.7", default-features = false }
-alloy-primitives = "0.8.14"
-alloy-provider = { version = "0.7", default-features = false, features = [
+alloy-json-rpc = { version = "0.9", default-features = false }
+alloy-network = { version = "0.9", default-features = false }
+alloy-node-bindings = { version = "0.9", default-features = false }
+alloy-primitives = "0.8"
+alloy-provider = { version = "0.9", default-features = false, features = [
     "reqwest",
     "ws",
 ] }
-alloy-pubsub = { version = "0.7", default-features = false }
-alloy-rlp = "0.3.9"
-alloy-rpc-client = "0.7"
-alloy-rpc-types = { version = "0.7", default-features = false, features = [
+alloy-pubsub = { version = "0.9", default-features = false }
+alloy-rlp = "0.3"
+alloy-rpc-client = "0.9"
+alloy-rpc-types = { version = "0.9", default-features = false, features = [
     "eth",
 ] }
-alloy-serde = { version = "0.7", default-features = false }
-alloy-signer = { version = "0.7", default-features = false }
-alloy-signer-aws = "0.7"
-alloy-signer-local = { version = "0.7", default-features = false }
+alloy-serde = { version = "0.9", default-features = false }
+alloy-signer = { version = "0.9", default-features = false }
+alloy-signer-aws = "0.9"
+alloy-signer-local = { version = "0.9", default-features = false }
 alloy-sol-types = "0.8"
-alloy-transport = { version = "0.7" }
-alloy-transport-http = { version = "0.7", features = ["reqwest-rustls-tls"], default-features = false }
-alloy-transport-ipc = { version = "0.7.0", default-features = false }
-alloy-transport-ws = { version = "0.7.0", default-features = false }
-alloy = { version = "0.7.0", features = ["sol-types", "contract","full","signer-aws"] }
+alloy-transport = { version = "0.9" }
+alloy-transport-http = { version = "0.9", features = ["reqwest-rustls-tls"], default-features = false }
+alloy-transport-ipc = { version = "0.9", default-features = false }
+alloy-transport-ws = { version = "0.9", default-features = false }
+alloy = { version = "0.9", features = ["sol-types", "contract","full","signer-aws"] }
```

Comment:
I think we can remove all the other `alloy` imports, since `alloy` just reexports all the other crates (e.g. `alloy_primitives` == `alloy::primitives`). I'll open an issue for this.

Reply:
https://github.com/Layr-Labs/eigensdk-rs/issues/197